CORE-DEVELOPMENT-README.md

```diff
@@ -0,0 +1,18 @@
+# CORE DEVELOPMENT GUIDE
+
+1. Clone this repository by using git clone https://github.com/bhanu951/drupal-core-development-project.git
+2. cd repos
+3. git clone --branch=10.1.x https://git.drupalcode.org/project/drupal.git drupal
+4. ddev composer install
+5. ddev composer require drupal/admin_toolbar drupal/devel
+6. ddev phpcs repo/drupal/core/<PATH_TO_FILE> (from project root)
+7. cd repos/drupal ; git status to track core changes
```

Comment:
There's already something about this in the README, but feel free to expand on it if you feel it's necessary.

Reply:
sure

Reply:
Done, Moved to the main README